File: Easy/0671-Second Minimum Node In a Binary Tree/Brute Force.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, x):
#         self.val = x
#         self.left = None
#         self.right = None

class Solution:
    def findSecondMinimumValue(self, root: TreeNode) -> int:
        vals = set()
        nodes = [root]

        while nodes:
            curr = nodes.pop()
            vals.add(curr.val)
            if curr.left:
                nodes.append(curr.left)
                nodes.append(curr.right)

        vals.remove(min(vals))

        return min(vals) if vals else -1

File: Easy/0796-Rotate String/Brute Force.rs
impl Solution {
    pub fn rotate_string(a: String, b: String) -> bool {
        let mut a = a;

        if a.len() == b.len() {
            for _ in 0..=a.len() {
                if a == b {
                    return true;
                }

                let ch = a.remove(0);
                a.push(ch);
            }
        }

        false
    }
}
